src/components/Addbook.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default function Addbook() {
+  return (
+    <div>
+      <h2>Add a Book</h2>
+      <input
+        type="text"
+        placeholder="
+      title of book"
+      />
+      <input type="text" placeholder="Author of book" />
+    </div>
+  );
+}
```

Comment:
- According to the project requirements, this component should have a button to submit (the button wouldn't add the book yet, this is just a placeholder for now). Please fix this 🙏

Reply:
ok